models/migrations/v34.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 Gitea. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/go-xorm/xorm"
+)
+
+// ActionV34 describes the removed fields
+type ActionV34 struct {
+	ActUserName  string `xorm:"-"`
+	RepoUserName string `xorm:"-"`
+	RepoName     string `xorm:"-"`
+}
+
+// TableName will be invoked by XORM to customize the table name
+func (*ActionV34) TableName() string {
+	return "action"
+}
+
+func removeActionColumns(x *xorm.Engine) (err error) {
```

Comment:
I know, I made a note of it in the PR description. How does one remove columns with xorm?